code/modules/paperwork/paper_premade.dm

```diff
@@ -31,6 +31,45 @@
 	name = "paper- 'Holopad Notice'"
 	default_raw_text = "<B>Can't get any botanists at the table? Have you tried using the damn holopad?</B><BR>\n<BR>\nStep onto the pad, and interface with it<BR>\nthen make your dang ol' call!<BR>\n<BR>\nYou want to call \"Hydroponics\" to reach them."
 
+/obj/item/paper/guides/jobs/engineering/bsa
+	name = "Bluespace Artillery (BSA) Assembly Instructions"
+	default_raw_text = {"<b>Below is a quick guide on setting up your new Bluespace Artillery Cannon.</b>
+	<br>
+	Featuring the newest in Nanotrasen technology, the Bluespace Artillery Cannon is your surefire ticket to complete and total threat elimination.
+	<br>
+	<h1>1. Acquire necessary resources.</h1>
+	* Bluespace Artillery Bore
+	* Bluespace Artillery Fusor
+	* Bluespace Artillery Generator
+	* Bluespace Artillery Controls
+	* 26 cable pieces
+	* 20 iron
+	* 20 bluespace crystals (or synthetic)
+	* 5 quadratic capacitors
+	* 5 femto servos
+	* 1 glass sheet
+	<h1>2. Find suitable location</h1>
+	The BSA has a steep power requirement and functions best when it is on its own APC.  Keep other items in the room to a minimum. Ensure the BSA is placed somewhere where it will not shoot at another part of the station.  Make sure the room isn't easily accessible by someone who shouldn't be there.
```

Comment:
```suggestion
	The BSA has a steep power requirement and functions best when it is on its own APC; keep other machines in the room to a minimum. Ensure the BSA is placed at the edge of the station, so it will not shoot through another part of the station.  Make sure the room isn't easily accessible by someone who shouldn't be there.
```